apiml-tomcat-common/src/main/java/org/zowe/apiml/filter/AttlsHttpHandler.java

```diff
@@ -89,17 +89,17 @@ ServerHttpRequest updateCertificate(ServerHttpRequest request, HttpServletReques
         }
 
         var str = String.format("""
-            -----BEGIN CERTIFICATE-----
-            %s
-            -----END CERTIFICATE-----
-            """,
-            Base64.encodeBase64String(rawCertificate)
+                        -----BEGIN CERTIFICATE-----
+                        %s
+                        -----END CERTIFICATE-----
+                        """,
+                Base64.getEncoder().encodeToString(rawCertificate)
```

Comment:
The indentation is incorrect
```suggestion
            -----BEGIN CERTIFICATE-----
            %s
            -----END CERTIFICATE-----
            """,
            Base64.getEncoder().encodeToString(rawCertificate)
```